packages/field-plugin/helpers/vite/src/sandbox.ts

```diff
@@ -0,0 +1,64 @@
+import * as querystring from 'querystring'
+import { bold } from './utils/text'
+import { arrows } from './utils/arrows'
+import { load, fileExists, MANIFEST_FILE_NAME } from './manifest'
+import type { Manifest } from './manifest'
+
+export const SANDBOX_BASE_URL = `https://plugin-sandbox.storyblok.com/field-plugin`
+
+export type SandboxQueryParams = {
+  url: string
+  manifest: Manifest | null
+}
+
+export const buildQueryString = (params: SandboxQueryParams) => {
+  const queryParams: any = {
+    url: params.url,
+    manifest: JSON.stringify(params.manifest),
```

Comment:
should query parameters that are null appear in the URL? 

Manifest can be null right? That is why the question

Reply:
I asked myself about displaying `?manifest=null` or not in the URL too but I convinced myself it would, somehow, be a hint to developers to learn more about this new feature and also less code on our side. Although, I  don't have a strong opinion on keeping it or not. 🤔

Reply:
If it's `?manifest=null`, then the Container will try to `JSON.parse("null")`, right? I think we could omit `manifest` in case it's null.

Reply:
When `?manifest=null` the Container assumes a default value @eunjae-lee.

```
const defaultManifest = { options: [] }
const manifestQueryParam = withDefault(JsonParam, defaultManifest)

const [manifest] = useQueryParam<{
  options: FieldPluginOption[]
}>('manifest', manifestQueryParam)
```

Reply:
Adjusted for just adding the `manifest` param if it's defined @BibiSebi @eunjae-lee.

Manifest found:
<img width="1433" alt="Screenshot 2023-09-18 at 15 59 06" src="https://github.com/storyblok/field-plugin/assets/1240591/5bc18d30-73bb-49e4-8fa4-8009e48dd126">

Manifest not found:
<img width="1433" alt="Screenshot 2023-09-18 at 16 03 05" src="https://github.com/storyblok/field-plugin/assets/1240591/eacf739e-abf4-4e45-8a62-2faec4c6c1fa">